source/backend/api/Services/DispositionFileService.cs

```diff
@@ -74,5 +99,69 @@ private void ReprojectPropertyLocationsToWgs84(IEnumerable<PimsPropertyDispositi
                 }
             }
         }
+
+        private void MatchProperties(PimsDispositionFile dispositionFile, IEnumerable<UserOverrideCode> overrideCodes)
+        {
+            foreach (var dispProperty in dispositionFile.PimsPropertyDispositionFiles)
+            {
+                if (dispProperty.Property.Pid.HasValue)
+                {
+                    var pid = dispProperty.Property.Pid.Value;
+                    try
+                    {
+                        var foundProperty = _propertyRepository.GetByPid(pid);
+                        dispProperty.PropertyId = foundProperty.Internal_Id;
```

Comment:
this is missing the 
UpdateLocation(acquisitionProperty.Property, ref foundProperty, userOverrideCodes);
method that is in the acquisitionfileservice.

What that'll mean is that when a pre-existing property is added to a disposition file it will not be possible to update it's location on the map. Perhaps this is ok?

Reply:
Added